full-stack-inspiration-board/submit.md

```diff
@@ -190,3 +190,28 @@ The first slide must include the team name and each member's name. The presentat
 ##### !end-answer
 ### !end-challenge
 <!-- prettier-ignore-end -->
+
+## Reflect on Pair Programming
+
+<!-- prettier-ignore-start -->
+### !challenge
+* type: paragraph
+* id: 0a403185-cde2-4a5b-9bfb-7333ff3db9f9
+* title: Reflection
+##### !question
+
+Reflect on the pair programming experience.  Respond to the following prompts:
+
+- How was working in a pair similar or different to working individually?
```

Comment:
Just a personal nit, but when else will I have a chance to complain about it? "different to" sounds very British English to me. 🇬🇧

Feel free to `won't fix` this as I know it would require making the change everywhere for consistency.

```suggestion
- How was working in a pair similar to or different from working individually?
```

Reply:
All the files are already touched by this PR, so I'm good running through and updating that line XD